src/modules/uxrce_dds_client/dds_topics.yaml

```diff
@@ -52,7 +52,7 @@ publications:
 
   - topic: /fmu/out/transponder_report
     type: px4_msgs::msg::TransponderReport
- 
+
```

Comment:
Unnecessary change?

Reply:
seems like it but I believe it was an extra space there. Make format did not complain with it.